style.css

```diff
@@ -578,6 +1526,170 @@ body {
     margin-left: 40%;
     margin-top: 116px;
     transform: rotate(135deg);
+    cursor: pointer;
+  }
+
+  .order {
+    flex-direction: column;
+    padding-bottom: 25px;
+  }
+
+  .order-image__first,
+  .order-image__second,
+  .order-number,
+  .range-1 {
+    display: none;
+  }
+
+  .order-images {
+    background-image: url('./images/pinkbag-mob.svg');
+    background-position: center;
+    background-repeat: no-repeat;
+    min-height: 240px;
+    transform: translateY(-41px);
+    width: 100%;
+  }
+
+  .order-details {
+    padding-left: 29px;
+    padding-right: 27px;
+    padding-top: 0;
+    transform: translateY(-8px);
+  }
+
+  .order-content {
+    padding-left: 27px;
+    padding-right: 34px;
+    padding-top: 17px;
+    transform: translateX(-5px);
+    width: 320px;
+  }
+
+  .order__title {
+    font-size: 20px;
+    line-height: 20px;
+  }
+
+  .order__subtitle {
+    font-size: 10px;
+    line-height: 10px;
+    margin-top: 5px;
+    transform: translateX(2px);
+  }
+
+  .order__description {
+    font-size: 9px;
+    line-height: 13px;
+    margin: 0;
+    transform: translateX(2px);
+  }
+
+  .color-list {
+    margin-bottom: 22px;
+    margin-left: 4px;
+    margin-right: 10px;
+  }
+
+  .list__item {
+    margin-right: 9px;
+  }
+
+  .list__item_pink {
+    height: 29px;
+    width: 29.39px;
+  }
+
+  .list__item_grey,
+  .list__item_blue,
+  .list__item_green {
+    height: 23px;
+    width: 23.35px;
+  }
+
+  .order__price {
+    font-size: 20px;
+    line-height: 28px;
+    margin: 0;
+  }
+
+  .order__cart {
+    font-size: 13px;
+    line-height: 0px;
+    height: 37px;
+    margin-bottom: 25px;
+    padding-left: 13px;
+    transform: translateX(2px);
+    width: 260px;
+  }
+
+  .outfit {
+  	background: linear-gradient(to right, #ebe3d8 27.5%, #000 27.5% 38.5%, #ebe3d8 38.5%);
+    flex-direction: column;
+    padding-left: 11px;
+    padding-top: 34px;
+    width: 100%;
+  }
+
+  .outfit__img {
+    height: 110%;
+    width: 110%;
+  }
+
+  .outfit-details {
+    margin: 0;
+    width: 100%;
+  }
+
+  .outfit-content {
+    margin-left: 35px;
+    padding-left: 25px;
+    padding-top: 6px;
+    transform: translateY(-22px);
+    width: 95%;
+  }
+
+  .outfit__title {
+    font-size: 20px;
+    line-height: 20px;
+  }
+
+  .outfit__subtitle {
+    font-size: 10px;
+    line-height: 10px;
+    margin-bottom: 0;
+    margin-top: 8px;
+  }
+
+  .outfit__description {
+    font-size: 9px;
+    line-height: 13px;
+    margin-bottom: 21px;
+    transform: translateX(-3px);
+  }
+
+  .outfit__cart {
+    border: 1px solid #000000;
```

Comment:
Codacy found an issue: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5841754)